libs/brackets-server/embedded-ext/file-extension/main.js

```diff
@@ -292,39 +291,57 @@ define(function (require, exports, module) {
         }
     }
 
+    function handleDownloadFile() {
+        const selected = ProjectManager.getSelectedItem();
+
+        if (selected && selected.isFile) {
+            const projectId = PreferencesManager.getViewState("projectId");
+            const filePath = brackets.app.convertFilePathToServerPath(selected.fullPath, projectId);
+            const anchor = document.createElement("a");
```

Comment:
const type would be fine ?
anchor.herf seems to be modified.

Reply:
Yes. 'anchor' is the object.
Object is reference type. So we can change member value of the 'anchor'.
(https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Statements/const)